modules/constants.py

```diff
@@ -38,6 +40,16 @@ def _get_schema(path):
         "all_pages": {"fields": _get_schema("all_pages.json")},
         "all_requests": {"fields": _get_schema("all_requests.json")},
     },
+    "additional_bq_parameters": {
+        "all_pages": {
+            'timePartitioning': {'type': 'DAY', 'field': 'date', 'requirePartitionFilter': True},
+            'clustering': {'fields': ['client', 'is_root_page', 'rank']}
+        },
+        "all_requests": {
+            'timePartitioning': {'type': 'DAY', 'field': 'date', 'requirePartitionFilter': True},
+            'clustering': {'fields': ['client', 'is_root_page', 'is_main_document', 'type']}
+        },
+    },
```

Comment:
note: this is added to aid with table creation. I used the existing table configuration as a reference, but the BigQuery API can be referenced for more
https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#jobconfigurationload